ipaserver/install/bindinstance.py

```diff
@@ -975,6 +976,17 @@ def __setup_named_conf(self):
         named_fd.write(named_txt)
         named_fd.close()
 
+        if not os.path.exists(paths.NAMED_CUSTOM_CONFIG):
+            ipa_ext_txt = ipautil.template_file(
+                os.path.join(paths.USR_SHARE_IPA_DIR, 'bind.ipa-ext.conf'),
+                {})
+            with open(paths.NAMED_CUSTOM_CONFIG, 'w') as ipa_ext:
+                ipa_ext.write(ipa_ext_txt)
+
+            pent = pwd.getpwnam(self.service_user)
+            os.chown(paths.NAMED_CUSTOM_CONFIG, pent.pw_uid, pent.pw_gid)
+            os.chmod(paths.NAMED_CUSTOM_CONFIG, 0o400)
```

Comment:
I propose to use `-rw-r-----  root named`

Reply:
on its way then. Nothing strong against it.